salt/modules/boto_iam.py

```diff
@@ -740,8 +740,34 @@ def delete_group(group_name, region=None, key=None,
         return False
 
 
-def create_login_profile(user_name, password, region=None, key=None,
-                         keyid=None, profile=None):
+def get_login_profile(user_name, region=None, key=None, keyid=None, profile=None):
+    '''
+    Return current Login Profile attached to user, if the user exists and
+    has a profile attached, None otherwise.
+
+    CLI Example:
```

Comment:
Can you add a `.. versionadded:: Fluorine` tag to this new function?

Each arg/kwarg needs documentation as well here. Can you add that?